azure-webapp-maven-plugin/src/it/14-linux-zip/pom.xml

```diff
@@ -0,0 +1,43 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.azure.it</groupId>
+    <artifactId>linux-zip</artifactId>
+    <packaging>war</packaging>
+    <version>1.0-SNAPSHOT</version>
+    <properties>
+        <maven.build.timestamp.format>yyyyMMddHHmmssSSS</maven.build.timestamp.format>
+    </properties>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>@project.groupId@</groupId>
+                <artifactId>@project.artifactId@</artifactId>
+                <version>@project.version@</version>
+                <configuration>
+                    <authentication>
+                        <serverId>azure-auth</serverId>
+                    </authentication>
+                    <resourceGroup>maven-webapp-it-rg-14</resourceGroup>
+                    <appName>maven-it-app-${maven.build.timestamp}</appName>
+                    <region>westus</region>
+                    <linuxRuntime>tomcat 8.5-jre8</linuxRuntime>
```

Comment:
The recommended usage should be: 

- tomcat -> warDeploy
- Java SE -> zipDeploy